[0.028s] Invoking command in '/home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble /usr/bin/cmake /home/mechax/zyb/radar_station/src/my_msgss -DCMAKE_INSTALL_PREFIX=/home/mechax/zyb/radar_station/src/my_msgss/install/my_msgss
[0.035s] -- Found ament_cmake: 1.3.6 (/opt/ros/humble/share/ament_cmake/cmake)
[0.115s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.116s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.117s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.134s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.249s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.249s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[0.249s]   my_msgss --arguments-file
[0.249s]   /home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss/rosidl_adapter__arguments__my_msgss.json
[0.249s]   --output-dir
[0.250s]   /home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss/rosidl_adapter/my_msgss
[0.250s]   --output-file
[0.250s]   /home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss/rosidl_adapter/my_msgss.idls)
[0.250s]   returned error code 1:
[0.250s] 
[0.250s]   Error processing 'point[] data' of 'my_msgss/Points': ''point' is an
[0.250s]   invalid message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$''
[0.250s] 
[0.250s]   Traceback (most recent call last):
[0.250s] 
[0.250s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.250s]       return _run_code(code, main_globals, None,
[0.250s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.250s]       exec(code, run_globals)
[0.250s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.250s]       sys.exit(main())
[0.251s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.251s]       abs_idl_file = convert_to_idl(
[0.251s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.251s]       return convert_msg_to_idl(
[0.251s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.251s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.251s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[0.251s]       Type(type_string, context_package_name=pkg_name),
[0.251s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 277, in __init__
[0.251s]       super(Type, self).__init__(
[0.251s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.251s]       raise InvalidResourceName(
[0.251s] 
[0.251s]   rosidl_adapter.parser.InvalidResourceName: 'point' is an invalid message
[0.251s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.251s] 
[0.251s] Call Stack (most recent call first):
[0.252s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.252s]   CMakeLists.txt:14 (rosidl_generate_interfaces)
[0.252s] 
[0.252s] [0m
[0.252s] -- Configuring incomplete, errors occurred!
[0.252s] See also "/home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss/CMakeFiles/CMakeOutput.log".
[0.254s] Invoked command in '/home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss' returned '1': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble /usr/bin/cmake /home/mechax/zyb/radar_station/src/my_msgss -DCMAKE_INSTALL_PREFIX=/home/mechax/zyb/radar_station/src/my_msgss/install/my_msgss
